networkx/algorithms/centrality/tests/test_domirank.py

```diff
@@ -0,0 +1,330 @@
+import pytest
+
+import networkx as nx
+
+pytest.importorskip("numpy")
+pytest.importorskip("scipy")
+
+
+class TestDomirank:
+    def test_K5(self):
+        """DomiRank centrality: K5"""
+        G = nx.complete_graph(5)
+        b, _, converged = nx.domirank(G)
+        assert converged
+        b_answer = {
+            0: 0.7916655540466309,
+            1: 0.7916655540466309,
+            2: 0.7916655540466309,
+            3: 0.7916655540466309,
+            4: 0.7916655540466308,
+        }
+        for n in sorted(G):
+            assert b[n] == pytest.approx(b_answer[n], abs=5e-3)
+        nstart = {n: 1 for n in G}
+        b, _, converged = nx.domirank(G, comp_method="analytical")
```

Comment:
```suggestion
        b, _, converged = nx.domirank(G, method="analytical")
```

If you change the name, you should do so in the tests as well.